tests/e2e/logging/test_openshift-logging.py

```diff
@@ -136,23 +136,44 @@ def validate_project_exists(self, pvc_obj):
         This function checks whether the new project exists in the
         EFK stack
         """
-
         pod_list = get_all_pods(namespace='openshift-logging')
         elasticsearch_pod = [
             pod.name for pod in pod_list if pod.name.startswith('elasticsearch')
         ]
         elasticsearch_pod_obj = get_pod_obj(
             name=elasticsearch_pod[1], namespace='openshift-logging'
         )
-        projects = elasticsearch_pod_obj.exec_cmd_on_pod(
-            command='indices | grep project', out_yaml_format=True
+        project_index = elasticsearch_pod_obj.exec_cmd_on_pod(
+            command='indices', out_yaml_format=False
         )
-        logger.info(projects)
-        if pvc_obj.project.namespace in projects:
-            logger.info("The new project exists in the EFK stack")
+        project = pvc_obj.project.namespace
+
+        if project in project_index:
+            logger.info(f'The project {project} exists in the EFK stack')
+            for item in project_index.split("\n"):
+                if project in item:
+                    logger.info(item.strip())
+                    if 'green' in item.strip():
+                        logger.info(f'The project {project} is Healthy')
+                    else:
+                        logger.info(f'The project {project} is unhealthy')
```

Comment:
You are just logging unhealthy project, isn't this unexpected behavior?

Reply:
Better change it to warn or error than info

Reply:
@Shrivaibavi as discussed offline, We don't expect the project's health to be unhealthy, so please assert if the project is unhealthy.